examples/benchmarking/criterion/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_rust//rust:rust.bzl", "rust_benchmark")
+load("//benchmarking/private:bench_runner.bzl", example_bench_runner_test = "bench_runner_test")
+
+package(default_visibility = ["//visibility:public"])
+
+rust_benchmark(
+    name = "bench",
+    srcs = ["benches/fibonacci.rs"],
+    edition = "2018",
+    use_libtest_harness = False,
+    deps = [
+        "//benchmarking/criterion/raze:criterion",
+    ],
+)
+
+example_bench_runner_test(
```

Comment:
Could we remove the macro indirection here? Feels like it makes the example less clear

Reply:
Done 😄